src/AltinnCore/Runtime/Views/Instance/editSPA.cshtml

```diff
@@ -13,16 +13,11 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.8.0/js/bootstrap-datepicker.min.js"></script>
 
 <script>
-  var routePrefix = '';
-  if (window.location.origin.includes('altinn3.no') || window.location.origin.includes('altinn.studio')) {
-    routePrefix = '/runtime';
-  }
-
   function loadScript() {
     // var script = document.createElement('script');
     var service = window.location.pathname.split('/');
     // script.type = 'text/javascript';
-    // script.src = window.location.origin + routePrefix +'/api/runtimeresources/runtime.js';
+    // script.src = window.location.origin '/api/runtimeresources/runtime.js';
```

Comment:
Shouldnt we remove these commented lines?

Reply:
fixed